README.rst

```diff
@@ -33,3 +33,14 @@ Bug tracker
 ===========
 A bug tracker is available at
 https://github.com/dataflake/dataflake.fakeldap/issues
+
+Building and Deploying
+======================
+
+Requirements:
+  - A Packagecloud.io key with Environment variable set to it, such as package_cloud_key="packagecloudkey"
+  - A clean branch
```

Comment:
I don't think there's any real concept of a clean *branch*, what you mean to say here is a clean *worktree*.